src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceControllerTest.kt

```diff
@@ -664,52 +663,5 @@ class LicenceControllerTest {
 
     val aStatusUpdateRequest =
       StatusUpdateRequest(status = LicenceStatus.APPROVED, username = "X", fullName = "Jon Smith")
-
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏Was not being used in the test so removed.